packages/grid/x-data-grid/src/colDef/gridMultipleSelectOperators.ts

```diff
@@ -0,0 +1,54 @@
+import { GridFilterInputMultipleSelect } from '../components/panel/filterPanel/GridFilterInputMultipleSelect';
+import { GridFilterOperator } from '../models/gridFilterOperator';
+import { GridFilterInputMultipleMultipleSelect } from '../components/panel/filterPanel/GridFilterInputMultipleMultipleSelect';
+import { isObject } from '../utils/utils';
+
+const parseObjectValue = (value: unknown) => {
+  if (value == null || !isObject<{ value: unknown }>(value)) {
+    return value;
+  }
+  return value.value;
+};
+
+export const getGridMultipleSelectOperators = (): GridFilterOperator[] => [
+  {
+    value: 'contains',
+    getApplyFilterFn: (filterItem) => {
+      if (filterItem.value == null || filterItem.value === '') {
+        return null;
+      }
+      return ({ value }): boolean =>
```

Comment:
The parameters for filter functions have changed: https://next.mui.com/x/react-data-grid/filtering/customization/#create-a-custom-operator